Python/Product/PythonTools/PythonTools/Repl/PythonInteractiveEvaluator.cs

```diff
@@ -134,7 +134,17 @@ partial class PythonInteractiveEvaluator :
                 if (config == null) {
                     _analyzer = _serviceProvider.GetPythonToolsService().DefaultAnalyzer;
                 } else {
-                    _analyzer = new VsProjectAnalyzer(_serviceProvider, interpreterService.FindInterpreter(config.Interpreter.Id));
+                    Build.Evaluation.Project projectFile = null;
+                    var moniker = ProjectMoniker ??
+                        interpreterService.GetProperty(config.Interpreter.Id, "ProjectMoniker") as string;
```

Comment:
Brought it back down to 2 places, and if you can suggest a good place for it that doesn't involve exposing knowledge about a particular interpreter configuration provider (and also doesn't imply that only a certain provider can use it, or that all providers need to support it), let me know and I'll make the change after I get back.